isotc204-glossary/tc204-termbase.meta.yaml

```diff
@@ -0,0 +1,16 @@
+---
+header:
+  register-name: ISO/TC 204 Intelligent transport systems (ITS)
+  content-summary: A compound register containing terminology records that have been
+    published in ISO/TC 204-sponsored International Standards and Technical Specifications.
+  date-of-last-change: '2022-05-30'
```

Comment:
This is the date when [iso-14812 source XML file](https://github.com/metanorma/iso-14812/blob/main/reference-docs/sts/iso_std_iso_ts_14812_ed-1_v1_en.xml) is last updated. 

@ronaldtse Is this correct or do I need to use some other date?

Reply:
Yes this is correct.